src/pages/VideoPage/VideoPage.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { VideoSection } from "./components";
+import "./VideoPage.css";
+
+import { videos } from "../../backend/db/videos";
+import { VideoCardHorizontal } from "../../components";
+
+const VideoPage = () => {
+  return (
+    <div className="video-page content ">
+      <VideoSection />
+      <section className="related-videos-section">
+        <h2 className="heading text-center gutter-bottom-24">Related Videos</h2>
+        <div className="videos-wrapper">
+          {videos.map((video) => (
+            <VideoCardHorizontal key={`${video._id}vs`} {...video} />
+          ))}
```

Comment:
What is "vs" is doing in the key ?

Reply:
I am using the id of video, which will be common on video page and on explore page, so just to give a unique key, used vs(video section )